devicon.json

```
@@ -9520,6 +9520,29 @@
         "color": "#F05138",
         "aliases": []
     },
+    {
+        "name": "swiper",
+        "altnames": [],
+        "tags": [
+            "framework",
```

Correct me if I'm wrong, but it seems like swiper is a javascript based library that is a part of framework7. So that would make swiper a library, not a framework, right?
```suggestion
            "library",
            "javascript",
```